### ISSUE TITLE
Add --init command to CLI

### ISSUE THREAD

**Issue by adidahiya** [palantir/tslint#717]
similar to ESLint, this would generate a starter config file for you

**Comment by myitcv** [comment]
On a related not, would it also make sense to generate the documentation (e.g. `README` which lists all the core rules) and the `sample.tslint.json`?

Could have a build step that checks the checked in code has been re-generated (that way you keep the generated output as part of the repo)

Ensures that everything remains in sync.

**Comment by JKillian** [comment]
@myitcv I like that idea, would makes things easier. https://github.com/palantir/tslint/issues/674 and #629 ed to be figured out first here though.

**Comment by adidahiya** [comment]
@myitcv @JKillian that please file that as a separate issue. A very basic `--init` feature is simple to implement right now and not blocked on anything.

**Comment by JKillian** [comment]
:+1:  Moved idea to part of #830
